### DISCUSSION TITLE
Query as Legend

### DISCUSSION THREAD

**Discussion by Sineos** [grafana/grafana#43315] [Dashboards, Panels & Visualizations]
I'm having graphs that show the current electrical power in Watt. I would love to create a legend that allow me to display the energy consumption for the same time frame as legend.
This could be achieved by either using an integral as legend or by querying a second metrics that already contains the needed values. In any case it would need basically a query as legend value.
Hope you can consider adding such a feature. Would be kind of legend-ary 😉

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
